packages/app/src/components/Header/index.tsx

```diff
@@ -10,7 +10,7 @@ import { AppBar, Avatar, Box, Button, ButtonGroup, IconButton, Menu, MenuItem, T
 import { useRecoilValue, useRecoilState, useSetRecoilState } from 'recoil';
 import React, { useContext, useEffect, useState } from 'react';
 import localforage from 'localforage';
-import { getMultinetDataUrl, oAuth } from '../../atoms/authAtoms';
+import { oAuth } from '../../api/auth';
```

Comment:
IE you want a code block of imports containing all the ones from auth?

Reply:
It doesnt need to be a code block, but generally I try to keep imports of the same subfolder (ie: `api`) together in an import list. It's very possible that ESlint will have some issue with this, because they seemingly take issue with anything

Reply:
Addressed in 0feddae8cc725563b53ba2c70345e1a5da2553c8